pkg/providers/aws/provider_sim.go

```diff
@@ -42,17 +42,16 @@ const (
 
 type SimClient struct {
 	Model      *models.Model
-	Outputs    map[string](*[]types.InstanceTopology)
+	Outputs    map[string]([]types.InstanceTopology)
 	NextTokens map[string]string
 }
 
-func (client SimClient) DescribeInstanceTopology(ctx context.Context, params *ec2.DescribeInstanceTopologyInput, optFns ...func(*ec2.Options)) (*ec2.DescribeInstanceTopologyOutput, error) {
-
+func (client *SimClient) DescribeInstanceTopology(ctx context.Context, params *ec2.DescribeInstanceTopologyInput, optFns ...func(*ec2.Options)) (*ec2.DescribeInstanceTopologyOutput, error) {
 	// If we need to calculate new results (a previous token was not given)
 	givenToken := params.NextToken
 	if givenToken == nil {
 		// Refreshes the clients internal storage for outputs
-		client.Outputs = make(map[string](*[]types.InstanceTopology))
+		client.Outputs = make(map[string]([]types.InstanceTopology))
```

Comment:
nit: Technically, you could remove the () also.

```suggestion
		client.Outputs = make(map[string][]types.InstanceTopology)
```